src/server/tran_page_requests.hpp

```diff
@@ -56,19 +56,17 @@ enum class page_to_tran_request
   SEND_TO_PTS_LOG_PRIOR_LIST, /* response-less */
 };
 
-// requets from page server to page server to catchup
+// requests from page server to page server to catchup
 enum class follower_to_followee_request
 {
   // Reserve for responses
   RESPOND,
 
-  SEND_DUMMY, // TODO for test. remove it.
-
   // TODO SEND_DISCONNECT, /* response-less */
-  // TODO SEND_LOG_PAGES_FETCH, /* synchronously waiting for response */
+  SEND_LOG_PAGES_FETCH, /* synchronously waiting for response */
 };
 
-// requets from page server to page server to catchup
+// requests from page server to page server to catchup
 enum class followee_to_follower_request
```

Comment:
Nit
`ps_followed_to_ps_follower_request`

Reply:
followee is not followed. It's the counter part of follower.